### ISSUE TITLE
Problem: Exceptions spam console when using wizard forms

### ISSUE THREAD

**Issue by anbud** [EmurgoHK/emurpas#47]
Problem: The following exception spams the console when using wizard forms:
```
Exception in template helper: TypeError: Cannot read property 'singleType' of undefined
    at SimpleSchema.getQuickTypeForKey (http://localhost:3000/packages/modules.js?hash=47567af7f264fc378e932c984c1fff318a172095:4476:40)
    at Object.getInputType (http://localhost:3000/packages/aldeed_autoform.js?hash=51e89e15ba8f6061bd5bdff815c619d09399c305:2469:29)
    at Object.getTemplateName (http://localhost:3000/packages/aldeed_autoform.js?hash=51e89e15ba8f6061bd5bdff815c619d09399c305:6343:30)
    at http://localhost:3000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:3051:16
    at http://localhost:3000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:1715:16
    at http://localhost:3000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:3103:66
    at Function.Template._withTemplateInstanceFunc (http://localhost:3000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:3744:12)
    at http://localhost:3000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:3102:27
    at Object.Spacebars.call (http://localhost:3000/packages/spacebars.js?hash=547cf8e466d1d52603d19bd5f48fb5df184fd237:172:18)
    at http://localhost:3000/packages/aldeed_autoform.js?hash=51e89e15ba8f6061bd5bdff815c619d09399c305:6316:27
meteor.js?hash=c8108d734cc548d91539b054e14b613d0c4512cc:1010 Exception in template helper: TypeError: Cannot read property 'singleType' of undefined
    at SimpleSchema.getQuickTypeForKey (http://localhost:3000/packages/modules.js?hash=47567af7f264fc378e932c984c1fff318a172095:4476:40)
    at Object.getInputType (http://localhost:3000/packages/aldeed_autoform.js?hash=51e89e15ba8f6061bd5bdff815c619d09399c305:2469:29)
    at Object.afFieldInputContext (http://localhost:3000/packages/aldeed_autoform.js?hash=51e89e15ba8f6061bd5bdff815c619d09399c305:6383:30)
    at http://localhost:3000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:3051:16
    at http://localhost:3000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:1715:16
    at http://localhost:3000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:3103:66
    at Function.Template._withTemplateInstanceFunc (http://localhost:3000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:3744:12)
    at http://localhost:3000/packages/blaze.js?hash=a1ff2d6d5ecd59ee11e2ba260b8650a9d1140f59:3102:27
    at Object.Spacebars.call (http://localhost:3000/packages/spacebars.js?hash=547cf8e466d1d52603d19bd5f48fb5df184fd237:172:18)
    at http://localhost:3000/packages/aldeed_autoform.js?hash=51e89e15ba8f6061bd5bdff815c619d09399c305:6317:23
```
It doesn't seem to affect the functionality.

Solution: Find what's causing it and fix it.

**Comment by gazhayes** [comment]
ping @samuelralak

**Comment by samuelralak** [comment]
I maye have an idea whats going on here.

**Comment by Pagebakers** [comment]
```js
if (typeof AutoForm.Utility.getFieldDefinition(ss, field + '.$').type === 'object') {

}
```

Seems to do the trick :) Hopefully @aldeed will fix this soon

**Comment by gazhayes** [comment]
Hey @Pagebakers do you happen to be looking for freelance work?

**Comment by anbud** [comment]
I've just noticed that this one is fixed, probably in #168.

**Comment by Pagebakers** [comment]
> Hey @Pagebakers do you happen to be looking for freelance work?

Sorry I didn't catch your comment before. Currently not looking for any gigs, but thanks for asking :)
